src/Core/Scheduler/Providers/WMS_Provider.js

```diff
@@ -134,4 +152,18 @@ WMS_Provider.prototype.executeCommand = function executeCommand(command) {
     }
 };
 
+WMS_Provider.prototype.getColorTextures = function getColorTextures(tile, layer, targetLevel) {
+    if (tile.material === null) {
+        return Promise.resolve();
+    }
+    const promises = [];
+    const bcoord = tile.getCoordsForLayer(layer);
+
+    for (const coordWMTS of bcoord) {
```

Comment:
Why a WMS provider would want to return several image for a single tile?

Reply:
it's same reason that PM tileMatrixSet